File: src/game/level/world2/World2Level.java
package game.level.world2;

import game.entity.enemies.boss.Boss;
import game.entity.enemies.npcs.FlyingEnemy;
import game.entity.enemies.npcs.TankEnemy;
import game.entity.enemies.npcs.YellowBall;
import game.entity.models.Enemy;
import game.level.Level;
import game.level.StoryLevel;

public abstract class World2Level extends StoryLevel {
    @Override
    public int getWorldId() {
        return 2;
    }

    @Override
    public final int getMaxDestroyableBlocks() {
        return 15;
    }

    @Override
    public Boss getBoss() {
        return null;
    }
}

File: src/game/level/world2/World2Level.kt
package game.level.world2

import game.entity.enemies.boss.Boss
import game.level.StoryLevel

abstract class World2Level : StoryLevel() {
    override val worldId: Int
        get() {
            return 2
        }

    override val maxDestroyableBlocks: Int
        get() {
            return 15
        }

    override val boss: Boss?
        get() {
            return null
        }
}